codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostStandardRequestDecoder.java

```diff
@@ -558,7 +560,7 @@ private void parseBodyAttributes() {
                     if (read == '&') {
                         currentStatus = MultiPartStatus.DISPOSITION;
                         ampersandpos = currentpos - 1;
-                        setFinalBuffer(undecodedChunk.retainedSlice(firstpos, ampersandpos - firstpos));
+                        setFinalBuffer(undecodedChunk.copy(firstpos, ampersandpos - firstpos));
```

Comment:
hmm... retainedSlice(...) should be fine. This will do an extra memory copy which I think is not needed.